br/pkg/task/stream.go

```diff
@@ -1660,6 +1669,16 @@ func restoreStream(
 		return errors.Annotate(err, "failed to restore kv files")
 	}
 
+	if cfg.ExplicitFilter {
```

Comment:
can we extend to full restore as well to prevent user read/write to tables being restored?

Reply:
in my mind full restore is supposed to be in a fresh cluster and no traffic coming to it? releasing restoreMode is going to be much slower as we need to release them using many DDLs. We could open an issue to see the perf impact before committing to it.